Makefile

```diff
@@ -157,8 +157,22 @@ docker-push: ## Push the docker image
 bundle-build: ## Build the bundle image.
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
+##@ Docs
+
+docs: mdtoc ## Update the Readme
+	mdtoc -inplace README.md
+
+mdtoc: ## Download mdtoc binary if necessary
+	GO111MODULE=off go get sigs.k8s.io/mdtoc || true
+
 ##@ Utilities
 
+generate-protoc: ## Generate the protobuf files
+	buf generate
+
+lint-protoc: ## lint the protocol files
+	buf lint
+
```

Comment:
do these need to be called as part of `make lint` and `make generate`?

Reply:
Hm, lint, maybe. The generate is not needed. Only if you change profiles.proto.